src/cljs/main/broadfcui/page/workspace/data/import_data.cljs

```diff
@@ -64,7 +64,5 @@
         :on-done (fn [{:keys [success? xhr get-parsed-response]}]
                   (swap! state dissoc :loading? :file :file-contents)
                   (if success?
-                    (do
                       (swap! state assoc :upload-result {:success? true})
-                      ((:reload props) (.-responseText xhr)))
```

Comment:
Now that we've unwrapped this `do`, the `swap!` line should get kicked back 2 spaces.